src/main/java/me/moodcat/mood/Mood.java

```diff
@@ -101,6 +102,16 @@ public static VAVector createTargetVector(final List<String> moods) {
         return VAVector.average(actualMoods);
     }
 
+    /**
+     * Create a {@link VAVector} with random valence and arousal values.
+     *
+     * @return the random vector.
+     */
+    public static VAVector createRandomVector() {
+        Random random = new Random();
+        return new VAVector(random.nextDouble(), random.nextDouble());
```

Comment:
Valence and arousal are in the range of [-1, 1] which means this method will only return vectors in [0, 1]. The random number has to multiplied by 2 and then 1 substracted of.